circle.yml

```diff
@@ -0,0 +1,14 @@
+machine:
+  xcode:
+    version: 9.1
+  environment:
+    LANG: en_US.UTF-8
+dependencies:
+  override:
+    - pod install --project-directory=Example --repo-update
+  cache_directories:
+    - "Carthage"
```

Comment:
We can probably drop this and the `cache_directories` since we don't use Carthage on CI.

Reply:
When you say this "this" do you mean just the cache_directories part?

Reply:
Oh, I meant both :)

Reply:
The override of the `pod install` ? How so ? Well, I guess if we include the Pods in CI we don't nee that line ?

Reply:
Sorry, I'm not being clear. We need to remove the `cache_directories` (and by extension, `"Carthage"`). 

Separately, running `pod install` will trigger a `Specs` repo setup, so we need to avoid it.

Reply:
No worries, I understood the point :) Thanks !